### ISSUE TITLE
github.com port443..failed error

### ISSUE THREAD

**Issue by tahammert** [AchimPieters/ESP8266-HomeKit-Air-Quality-Sensor#1]
I want to run this code on my nodemcu, but after connecting to the wifi network and entering the ota addresses, the serial port says "ota_connect LocalPort=e0e8 DNS IP:140.82.121.3 local..OK remote..OK SSL..OK set_fd to github.com port 443.. failed, return [-0x1]
I get the error "wolfSSL_send error = -313". How do I solve this?
SerialPort:
![Ekran Görüntüsü (338)](https://github.com/AchimPieters/ESP8266-HomeKit-Air-Quality-Sensor/assets/72361022/d9b74b18-10d1-4e95-8c86-b036f62ef3c1)

**Comment by AchimPieters** [comment]
Use the latest LCM version found here:

https://github.com/HomeACcessoryKid/life-cycle-manager

**Comment by tahammert** [comment]
I replaced the "otaboot.bin" file with the newest one, and this time the "ota-parameter" section appeared in the wifi ap settings. I left this blank, specified the path to your github file, and joined the network. This time it doesn't give an error directly, it seems to progress a little, but as a result "main.bin" cannot be installed. I'm adding the serial port outputs again. I tried to transfer your "main.bin" file to nodemcu without using OTA and I think it goes into a ridiculous loop because it cannot join the wifi network. Is there any other way to do this? or how can I fix this error?

![Ekran Görüntüsü (341)](https://github.com/AchimPieters/ESP8266-HomeKit-Air-Quality-Sensor/assets/72361022/16f01528-3607-49dd-9816-ac4e09a97e2c)
![Ekran Görüntüsü (340)](https://github.com/AchimPieters/ESP8266-HomeKit-Air-Quality-Sensor/assets/72361022/2a8f3aab-b1f8-4211-b5cf-c10077ba1a16)
